improve description in fastlane

IzzySoft opened the pull request:
This PR improves short and full description in Fastlane, to give people a better idea of what the app is about. Helpful for your pending RFP at F-Droid – as well for the [completed one at IzzyOnDroid](https://gitlab.com/IzzyOnDroid/repo/-/issues/848), where Squeezelite will become available with the next sync around 6 pm UTC; be welcome to [pick a badge](https://gitlab.com/IzzyOnDroid/repo/#if-my-app-is-available-here-do-you-have-a-badge-i-can-use-to-link-to-it) and link to `https://apt.izzysoft.de/packages/org.lyrion.squeezelite` e.g. from your Readme. As the app was also confirmed to be reproducible, there's also a shield to show its latest RB status, if you want it:

[<img src="https://shields.rbtlog.dev/simple/org.lyrion.squeezelite" alt="RB Status">](https://shields.rbtlog.dev/org.lyrion.squeezelite)

`[<img src="https://shields.rbtlog.dev/simple/org.lyrion.squeezelite" alt="RB Status">](https://shields.rbtlog.dev/org.lyrion.squeezelite)`

IzzySoft committed:
improve description in fastlane

bb010g commented on fastlane/metadata/android/en-US/full_description.txt:
```diff
@@ -1 +1 @@
-'Headless' audio player for Lyrion Music Server (LMS) - Squeezelite ported to Android.
+<p><i>Squeezelite</i> is a 'headless' audio player for Lyrion Music Server (LMS) - <a href='https://github.com/LMS-Community/slimserver'>Squeezelite</a>, ported to Android. You will need an LMS server in your local network to make use of this; <a href='https://lyrion.org/getting-started/'>see here</a> for getting started.</p>
```
```suggestion
<p><i>Squeezelite</i> is a 'headless' audio player for Lyrion Music Server (LMS) - <a href='https://github.com/LMS-Community/slimserver'>Squeezelite</a>, ported to Android. You will need access to an LMS server from your network to make use of this; see <a href='https://lyrion.org/getting-started/'>Getting Started with LMS</a>.</p>
```

IzzySoft committed:
Update fastlane/metadata/android/en-US/full_description.txt

Thanks! That was the part I was not 100% sure (which of the two projects was meant).

Co-authored-by: bb010g <me@bb010g.com>

IzzySoft commented:
Thanks! That was the part I wasn't sure about (which of the two projects were meant). Sorry for messing up the commit message, thought that was meant as comment :see_no_evil: Feel free to squash for a clean history…

IzzySoft commented:
Thanks for merging and adding the shield to the Readme! Will you take the badge there as well?

[<img src="https://gitlab.com/IzzyOnDroid/repo/-/raw/master/assets/IzzyOnDroidButtonGreyBorder_nofont.png" height="80" alt="Get it at IzzyOnDroid">](https://apt.izzysoft.de/packages/org.lyrion.squeezelite)

`[<img src="https://gitlab.com/IzzyOnDroid/repo/-/raw/master/assets/IzzyOnDroidButtonGreyBorder_nofont.png" height="80" alt="Get it at IzzyOnDroid">](https://apt.izzysoft.de/packages/org.lyrion.squeezelite)`

(if you prefer one of the other badged, there's a small collection, e.g. NoBorder: `IzzyOnDroidButton.png`, or the old original one with border and spacing (matching the "official" ones by F-Droid and Play): `IzzyOnDroid.png`)

CDrummond commented:
Done :)

IzzySoft commented:
:star_struck: Thanks!
